### ISSUE TITLE
Step 6: Finalize Aug 2023 report

### ISSUE THREAD

**Issue by JLWade** [bcgov/parks-reso-public#353]
#### Description:

This ticket is to finalize the Aug 2023 DUP parkinfo email report. 

#### Acceptance Criteria:

- [x] Add take-aways identified in #352 to the report started in #351 (align with take-aways to correct categories)
- [x] Add relevant charts and visuals to this report from #354 
- [x] Add quotes to this report from #355 
- [x] Add small purpose at the beginning (similar to July 2023)
- [ ] Add brief initial findings to this report
- [x] Shared with Diana for review and edits
- [x] Shared with Jess for review and edits

#### Development Checklist:

- utilize July 2023 to guide sections and scope
- do not include "introduction," "Background" or "high level project details" (already covered in July 2023)

**Dependencies**
- access to emails
- blocked by #350 #351 #352 #354 #355 

**Definition of Ready**

- [x] Acceptance criteria are included
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been reviewed and questions answered

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are complete
- PO Review:
  - [ ] Acceptance criteria are confirmed complete
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by kgorana** [comment]
@JLWade  I haven't added a brief of initial findings - not sure if it still fits as a separate section in the report, but I'm happy to do that, so let me know.
